fullstack-projects/url-shortener/devashmit-day8/backend/database.py

```
@@ -0,0 +1,19 @@
+    with get_conn() as c:
+        c.execute("""CREATE TABLE IF NOT EXISTS urls(
+            code TEXT PRIMARY KEY,
+            original TEXT NOT NULL,
+            clicks INTEGER NOT NULL DEFAULT 0,
+            created_at TEXT NOT NULL DEFAULT (datetime('now')))""")
+
+@contextmanager
+def get_conn():
+    c = sqlite3.connect(DB)
+    c.row_factory = sqlite3.Row
+    try: yield c; c.commit()
```

Ruff (0.15.15)

[error] Multiple statements on one line (colon)

(E701)